README.md

```diff
@@ -162,79 +161,71 @@ listed [above](#requirements), including having an AMI suitable for running
+should be layered on top of the associated infrastructure configuration.
 
-The following sections walk through instantiating the `overlay` workload.
-More details on all the Terraform command options can be found in the
-[Terraform CLI documentation](https://developer.hashicorp.com/terraform/cli).
+The following sections walk through instantiating the Overlay example.  More
+details on all the Terraform command options can be found in the [Terraform CLI
+documentation](https://developer.hashicorp.com/terraform/cli).
 
 ### Bring-up
 
 Firstly, clone this repository.
 
-The `eks-cluster` module must be run first; this provisions a minimal EKS
-cluster.
-
-Use [`terraform
-init`](https://developer.hashicorp.com/terraform/cli/commands/init) and
-[`terraform
-apply`](https://developer.hashicorp.com/terraform/cli/commands/apply) to run
-the `eks-cluster` module:
+The Bootstrap configuration must be run first; this provisions a VPC, EKS
+cluster, and Bastion node (for worker node access).
 
 ```
-terraform -chdir=examples/infra/eks-cluster init
-terraform -chdir=examples/infra/eks-cluster apply
+terraform -chdir=examples/bootstrap init
+terraform -chdir=examples/bootstrap apply
 ```
 
 Terraform will show you a changeset and ask you to confirm that it should
-proceed.  It takes around 15 minutes to bring up the EKS cluster.
+proceed.  It takes around 15 minutes to bring up the configuration.
 
-Then run the `eks-setup` module to setup the EKS cluster:
+Then apply the Overlay infrastructure configuration.  This creates the
+remaining cloud infrastructure resources necessary for running the Overlay
+workload.
 
 ```
-terraform -chdir=examples/infra/eks-setup init
-terraform -chdir=examples/infra/eks-setup apply
+terraform -chdir=examples/overlay/infra init
+terraform -chdir=examples/overlay/infra apply
 ```
 
-When the cloud infrastructure modules have been applied, you can then apply
-workload modules.  The procedure is largely the same; although you should first
-check the [`variables.tf`](examples/overlay/variables.tf) file to find all the
-configuration options, and then create a file `vars.tfvars` to set the
-configuration options you want use following the [variable definitions file
-format](https://developer.hashicorp.com/terraform/language/values/variables#variable-definitions-tfvars-files):
+Finally apply the Overlay workload configuration.  This accepts a number of
+input variables described in
+[`variables.tf`](/examples/overlay/workload/variables.tf) which may be of
+interest; e.g., it is necessary to set the IOS XR root username and password.
+To do so, create a file `vars.tfvars` with the desired configuration options
+following the [variable definitions file
+format](https://developer.hashicorp.com/terraform/language/values/variables#variable-definitions-tfvars-files).
 
 ```
 cat << EOF
-cluster_version = 1.28
 xr_root_user = "root"
 xr_root_password = "mypassword"
 EOF > vars.tfvars
```

Comment:
This seems to not works, should be  
```
cat << EOF > vars.tfvars
xr_root_user = "myusername"
xr_root_password = "mypassword"
EOF
```
(Also a different username instead of root as above.)